tslint.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I believe TSLint is deprecated now and ESLint with the TypeScript plugin should be used.

Reply:
Do you by any chance have an example of a project using it?

Reply:
gonfalon uses it. You can see how to implement it in the .eslintrc.js file.